app/routes-config.js

```diff
@@ -4,5 +4,9 @@ export const profile_view = id => `/users/${id}`;
 export const profile_edit = id => `/users/edit/${id}`;
 export const questions = () => `/questions`;
 export const question_view = id => `/questions/${id}`;
+export const question_edit = (user, link, questionid) =>
```

Comment:
Let's try to format routes close to REST notation, as it is easier to read and understand.
Edit question URL should be: `questions/:questionId/edit`
Edit answer URL should be `questions/:questionId/answers/:answerId/edit`

I was thinking about link and user parameters in URL. It seems as bad practice to me. Let's load them from blockchain using questionId and answerId.